src/resources/js/components/CreatePoll.vue

```diff
@@ -1,65 +1,52 @@
 <template>
-  <v-form
-    ref="form"
-    v-model="valid"
-    lazy-validation
-  >
-    <v-text-field
-      v-model="input.subject"
-      :rules="subjectRules"
-      label="Thème"
-      required
-    />
-
-    <v-btn
-      :disabled="!valid"
-      color="success"
-      class="mr-4"
-      @click="createPoll"
-    >
-      Créer sondage
-    </v-btn>
-
-  </v-form>
+  <v-card>
+      <v-card-title>
```

Comment:
indentation not the same T_T
(the line just above as a 2 space tab and the rest has 4)

Reply:
fixed last commit